modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -131,28 +115,425 @@ on [.ui]*travel-sample*, the view is as follows:
+
+The *Users & Groups* screen now appears as follows:
+
+image::manage-security/usersAndGroupsTwoUsers.png[,800,align=middle]
+
+The new user is duly shown to have the `Full Admin` role associated with the the `Admins` group.
+
+=== Add and Edit Groups
+
+The step-by-step procedures for adding and editing groups are provided in xref:manage:manage-security/configure-ldap.adoc[Configure LDAP].
 
 === Role-Based Console Appearance
 
 Role-assignment determines which features of Couchbase Web Console are
 available to the administrator.
 Non-available features are not displayed: therefore, the console's appearance
-changes, based on which roles have been assigned to the current user.
+changes, based on which roles have been assigned the current user.
+
+[#manage-users-with-the-cli]
+== Manage Users with the CLI
+
+Users can be managed with the xref:cli:cli-user-manage.adoc[user-manage] command.
+This allows the creation and deletion of users and groups, the assignment of roles, and the listing of current status.
+
+[#get-user-information-with-the-cli]
+=== Get User Information with the CLI
+
+To list the cluster's current users, enter the following.
+Note that the command is piped to the `jq` program, to optimize output-readability.
+
+----
+/opt/couchbase/bin/couchbase-cli user-manage --cluster http://10.143.192.101 \
+--username Administrator \
+--password password \
+--list | jq
+----
+
+A document is returned, containing an entry for each of the current users:
+
+----
+[
+  {
+    "id": "externalUserIndividual",
+    "domain": "external",
+    "roles": [
+      {
+        "role": "cluster_admin",
+        "origins": [
+          {
+            "type": "user"
+          }
+        ]
+      }
+    ],
+    "groups": [],
+    "external_groups": [],
+    "name": "John Smith"
+  },
+  {
+    "id": "localUser",
+    "domain": "local",
+    "roles": [
+      {
+        "role": "data_writer",
+        "bucket_name": "travel-sample",
+        "origins": [
+          {
+            "type": "user"
+          }
+        ]
+      },
+      {
+        "role": "data_reader",
+        "bucket_name": "travel-sample",
+        "origins": [
+          {
+            "type": "user"
+          }
+        ]
+      }
+    ],
+    "groups": [],
+    "external_groups": [],
+    "name": "",
+    "password_change_date": "2019-05-31T03:59:49.000Z"
+  },
+  {
+    "id": "externalUserGroup",
+    "domain": "external",
+    "roles": [
+      {
+        "role": "admin",
+        "origins": [
+          {
+            "type": "group",
+            "name": "Admins"
+          },
+          {
+            "type": "user"
+          }
+        ]
+      }
+    ],
+    "groups": [
+      "Admins"
+    ],
+    "external_groups": [],
+    "name": "David Brown"
+  }
+]
+----
+
+The entries include information on the `name` and `roles` of the user, and on the local and external `groups` to which the user belongs.
+
+Information on currently defined _groups_ can similarly be returned:
```

Comment:
I guess the example doesn't match the description here. List of defined groups can be retrieved by
```
./install/bin/couchbase-cli user-manage --cluster http://10.143.192.101 \
--username Administrator \
--password password \
--list-groups
```

Reply:
Thanks, I've fixed this.